tests/testing.md

```diff
@@ -151,6 +160,16 @@ See
     > **NOTE**: This command will prompt for sudo permission
     > on systems where 80 and 443 are privileged ports
 
+1. Copy the 'Voxel51 GitHub Internal' license file and convert it to a
```

Comment:
This won't work for our integration tests (that we unfortunately don't have a PR check for).  Each test case run creates a unique ephemeral namespace. I'm working on a solution for this, but it doesn't need to block this PR.

Reply:
Yeah. I couldn't find a way to make that work dynamically...